frontend/src/app/components/routes/main-view/components/meetings/meetings.component.html

```diff
@@ -12,6 +12,7 @@
                     mat-raised-button
                     class="button--light"
                     (click)="onNewMeetingClick()"
+                    [ngClass]="{ 'button--disabled': isLoading$ | async }"
```

Comment:
klase z scss da się globalnie dac buttonowi, żeby nie tzreba było w każdym komponencie jej dodawać

Reply:
Tzn dać buttowi, że jeśli jest disabled to ma mieć określone style